core/src/commonMain/kotlin/com/xebia/functional/xef/llm/models/functions/JsonSchema.kt

```diff
@@ -111,7 +111,7 @@ fun encodeJsonSchema(descriptor: SerialDescriptor): String =
   Json.encodeToString(JsonObject.serializer(), buildJsonSchema(descriptor))
 
 /** Creates a Json Schema using the provided [descriptor] */
-private fun buildJsonSchema(descriptor: SerialDescriptor): JsonObject {
+fun buildJsonSchema(descriptor: SerialDescriptor): JsonObject {
```

Comment:
Should we remove `encodeJsonSchema` since it's not used now?

Reply:
Removed, thanks